### PR TITLE
feat(users): add database, resolvers, and queries for users

### PR DESCRIPTION
**Pull request by Edholm**
## Description

Add a database table for users, a `me` query that inserts or selects the current user.

## How has this been tested?

- [ ] Unit tests
- [x] Manually

### CODE REVIEW

File: api/graphql/v2/users.graphqls [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    id: UUID!
+    filmstadenMembershipId: String
+    name: String!
+    firstName: String!
+    lastName: String!
+    nick: String
+    email: String!
+    phone: String
+    avatarURL: String
+#    giftCertificates: [GiftCertificateDTO!]!
```
Comment (ericalofstrom) [inline api/graphql/v2/users.graphqls:18]: Check: How come these are ignored for now?

File: configs/database/migrations/000003_create_users.down.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS users;
```
Comment (ericalofstrom) [inline configs/database/migrations/000003_create_users.down.sql:1]: Check: Will all users be required to be recreated?

File: internal/database/sqlc/models.go [hunk 3]
```diff
@@ -3,11 +3,30 @@
...
 
 type Commandment struct {
 	Number      int32     `json:"number"`
 	Phrase      string    `json:"phrase"`
 	CreatedDate time.Time `json:"createdDate"`
 }
+
+type User struct {
+	ID                     uuid.UUID      `json:"id"`
+	SubjectID              string         `json:"subjectID"`
```
Comment (ericalofstrom) [inline internal/database/sqlc/models.go:20]: Question: What's the difference between string and sql.NullString?

**Review by ericalofstrom — APPROVED** [review]

File: configs/database/migrations/000003_create_users.down.sql (hunk 2 above)
Comment (Edholm) [reply configs/database/migrations/000003_create_users.down.sql:1]: yes

File: api/graphql/v2/users.graphqls (hunk 1 above)
Comment (Edholm) [reply api/graphql/v2/users.graphqls:18]: not implemented yet (requires another table)

File: internal/database/sqlc/models.go (hunk 3 above)
Comment (Edholm) [reply internal/database/sqlc/models.go:20]: `sql.NullString` is the way go has chosen to represent a nullable database string. `*string` could've worked just as well, but they didn't go that route for some reason.

Comment (Edholm) [reply internal/database/sqlc/models.go:20]: yes, but it was still a legit question.

**Pull request merged**